src/Pages/SignIn/SignIn.js

```diff
@@ -47,9 +52,26 @@ const SignIn = ({ loginStatus }) => {
   };
 
   const handleLogin = () => {
-    if (loginStatus) {
-      setLogin(true);
-    }
+    axios({
+      method: "POST",
+      url: `${API}hrdsuser/login/`,
+      data: {
+        email: inputUserName,
+        password: inputPassword,
+      },
+    })
+      .then((res) => {
+        sessionStorage.setItem("access_token", res.data.data.token);
+        setLogin(true);
+      })
+      .then(() => {
+        history.push("/");
+      });
+  };
+
+  const handleInput = (e) => {
+    e.target.name === "email" && setUserName(e.target.value);
+    e.target.name === "password" && setPassword(e.target.value);
```

Comment:
```suggestion
  const [form, setValues] = useState({
    email: "",
    password: "",
  });
  
  const handleInput = ({ target: { name, value } }) => {
    setValues({
      ...form,
      [name]: value,
    });
  };
```
이렇게 하고 각 input의 value에 value={form.email} value={form.password} 이런식으로도 됩니다.

Reply:
[대괄호 안에 변수] 넣으면 키로 쓸 수 있는 것도 지금 알았어요 ! 감사합니다 👍🏻